doc/api/esm.md

```diff
@@ -1242,11 +1254,18 @@ export async function getFormat(url, context, defaultGetFormat) {
 }
 ```
 
-#### <code>getSource</code> hook
+#### `getSource(url, context, defaultGetSource)`
 
 > Note: The loaders API is being redesigned. This hook may disappear or its
 > signature may change. Do not rely on the API described below.
 
+* `url` {string}
+* `context` {Object}
+  * `format` {string}
+* `defaultGetSource` {Function}
+* Returns: {Object}
+  * `source` {string|SharedArrayBuffer|Uint8Array}
```

Comment:
Same question as above —  is this supposed to be documented as an object wrapped in a promise (compare the JSDoc below)?